docs/fx-core/spfx-help.md

```diff
@@ -116,4 +117,39 @@ Install the SPFx generator version that Teams Toolkit supports, say `1.14`:
 
 ```sh
 npm install @microsoft/generator-sharepoint@1.14 -g
-```
\ No newline at end of file
+```
+
+## 3. Failure to add additional SPFx tab<a name="addfeature"></a>
+
+Multi-tab SPFx project is supported in Teams Toolkit. To add an additional SPFx tab, you can try "Add features" and select "SPFx tab". Behind the scenes, Yeoman generator is executed according to configuration file(.yo-rc.json) in current solution.
+
+If .yo-rc.json file doesn't exist in your SPFx project, adding SPFx tab might fail with potential issues. 
```

Comment:
If .yo-rc.json file doesn't exist in your SPFx project, adding SPFx tab will fail with the following error: